libraries/AP_HAL_ChibiOS/hwdef/CUAV-Nora/hwdef-bl.dat

```diff
@@ -0,0 +1,58 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H743 bootloader
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# USB setup
+USB_VENDOR 0x0483 # ST
+USB_PRODUCT 0x5740
+USB_STRING_MANUFACTURER "ArduPilot"
+USB_STRING_PRODUCT "%BOARD%"
+USB_STRING_SERIAL  "%SERIAL%"
+
+# board ID for firmware load
+APJ_BOARD_ID 1009
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 0
+
+# the location where the bootloader will put the firmware
+# the H743 has 128k sectors
+FLASH_BOOTLOADER_LOAD_KB 128
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+PD8 LED_RED OUTPUT OPENDRAIN HIGH # red
+PC7 LED_ACTIVITY OUTPUT OPENDRAIN HIGH # blue
+PC6 LED_BOOTLOADER OUTPUT OPENDRAIN HIGH # green 
+
+define HAL_LED_ON 0
+
+# board voltage
+STM32_VDD 330U
+
+# order of UARTs (and USB)
+UART_ORDER OTG1 UART7
+
+# UART7 is debug
+PF6 UART7_RX UART7 NODMA
+PE8 UART7_TX UART7 NODMA
+
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+define HAL_USE_EMPTY_STORAGE 1
+define HAL_STORAGE_SIZE 32768
```

Comment:
should be 16384 till we add support for larger sizes

Reply:
OK, I'll wait to modify it, because 32768 can be used and tested